server/config/routes.js

```diff
@@ -1218,4 +1219,11 @@ exports.configure = function configure(app) {
   app.post('/shipment_containers', shipmentContainer.create);
   app.put('/shipment_containers/:uuid', shipmentContainer.update);
   app.delete('/shipment_containers/:uuid', shipmentContainer.deleteContainer);
+
+  // funding sources
+  app.get('/funding_sources', fundingSource.read);
```

Comment:
Nice API design!  Is there a vision to use these funding sources outside of stock management?

Otherwise, we might put them under the `/stock/` prefix.

Reply:
Not only in stock